tsdef/AudioTrack.d.ts

```diff
@@ -9,7 +9,7 @@ export class AudioTrack extends Track {
   attach(element?: HTMLMediaElement | string): HTMLMediaElement;
   detach(element?: HTMLMediaElement | string): HTMLMediaElement[];
 
-  on(event: 'disabled', listener: (track: this) => void): this;
-  on(event: 'enabled', listener: (track: this) => void): this;
-  on(event: 'started', listener: (track: this) => void): this;
+  on(event: 'disabled', listener: (track: this) => any): this;
```

Comment:
Just curious, why is `void` being changed to `any`? This is nit-picky, but I think `void` is correct. This is consistent with how event listeners are defined in the node.js type definitions.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v12/events.d.ts#L33

Reply:
This was updated to `void` to be consistent with how RTC Diagnostics implements their event listeners. @mhuynh5757 pointed out to me that event listeners should not be opinionated. 

https://github.com/twilio/rtc-diagnostics/blob/master/lib/AudioInputTest.ts#L90

Reply:
I see. I don't think it makes any real difference whether we use `any` or `void`, but I'd personally lean towards keeping things consistent with the types used in the `EventEmitter` class since these classes are inheriting from it.

Also, the TS Docs do say to use `void` instead of `any`.

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#return-types-of-callbacks

Reply:
@mhuynh5757 @manjeshbhargav 

@timmydoza I hear you. I'm also open to more input and discussion as well on which to use. I think if we go with `void`, then should we not be doing that for RTC Diagnostics as well? I'd like to aim for consistency, I think.

Reply:
Thanks for bringing that up @timmydoza , didn't know that TS prefers `void` over `any`. Makes sense 👍